charts/kube-acp-stack/values.yaml

```diff
@@ -40,3 +40,9 @@ redis-cluster:
     extraVolumeMounts:
       - mountPath: /redismodules
         name: redismodules
+timescaledb-single:
+  enabled: true
+  clusterName: timescale
```

Comment:
do we need set this one?

Reply:
yes. if this is not set, timescale will use release name as default name for service. On the other hand, ACP also uses release name as default name for its service and this create conflict. To avoid the conflict this config param needs to set.